client/client_test.go

```diff
@@ -60,18 +60,18 @@ func TestAcquire(t *testing.T) {
 	}{
 		{
 			name:      "request error",
-			serverErr: "Acquire failed: no available resource my-resource-type, try again later",
```

Comment:
isn't `Acquire failed: ` part of the error message from the server?

Reply:
Yeah - that's a good catch - I switch from an exact string match to doing a substring compare because the message returned from the server is actually a composite of the straight error message and the constant string "Acquire failed:"